src/jit/lir.cpp

```diff
@@ -129,7 +129,14 @@ GenTree* LIR::Use::User() const
 //
 bool LIR::Use::IsValid() const
```

Comment:
I'd say it the other way around: the IsValid() checks subsume the ctor checks, which can now just call IsValid()